extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilderController.class.php

```diff
@@ -69,6 +69,17 @@ public function getRedirectUrls() {
 		}
 	}
 
+	public function checkIfTemplateExists() {
+		$status = new Status();
+		$response = $this->getResponse();
+		$response->setFormat( WikiaResponse::FORMAT_JSON );
+		$requestParams = $this->getRequest()->getParams();
+		$title = PortableInfoboxBuilderHelper::getTitle( $requestParams[ 'title' ], $status );
```

Comment:
You could use $this->getRequest->getVal('title') without creating the supporting value